components/03-modules/catalog-grid-item/catalog-grid-item.hbs

```diff
@@ -47,13 +47,14 @@
                 {{ render '@button' primaryAction }}
             </form>
 
-            <button class="catalog-grid-item__secondary-action">
-                {{ render '@icon' secondaryActions.0.icon }}
-            </button>
-
-            <button class="catalog-grid-item__secondary-action">
-                {{ render '@icon' secondaryActions.1.icon }}
-            </button>
+            <div class="catalog-grid-item__secondary-action">
+                <button class="button button--icon">
+                    {{ render '@icon' secondaryActions.0.icon }}
+                </button>
+                <button class="button button--icon">
+                    {{ render '@icon' secondaryActions.1.icon }}
+                </button>
```

Comment:
Can we use button--icon component instead of button tags and icons?

Reply:
Done